run-checks

```diff
@@ -84,6 +84,39 @@ append_coverage() (
     fi
 )
 
+missing_interface_spread_test() {
+    snap_yaml="tests/lib/snaps/test-snapd-policy-app-consumer/meta/snap.yaml"
+    core_snap_yaml="tests/lib/snaps/test-snapd-policy-app-provider-core/meta/snap.yaml"
+    classic_snap_yaml="tests/lib/snaps/test-snapd-policy-app-provider-classic/meta/snap.yaml"
+    for iface in `ls -1 ./interfaces/builtin/ | grep -Ev '_test\.go' | sed -e 's/\.go$//' -e 's/_/-/g'` ; do
```

Comment:
Just a thought bu maybe it would be better to use a tiny Go program like this one instead of transforming file names:
```
import "fmt"
import "github.com/snapcore/snapd/interfaces/builtin"

func main() {
	for _, iface := range builtin.Interfaces() {
		fmt.Printf("%s\n", iface.Name())
	}
}
```
You can put it under `tests/lib/listbuiltin.go` and run with `go run tests/lib/listbuiltin.go`

Even better would be to verify spread yamls directly in Go.

Reply:
Done (with list-interfaces.go)